src/styles/partials/_jumbo.scss

```diff
@@ -9,39 +9,39 @@
   background-position-y: 10vh;
 
   @include mq($until: sm) {
-    background-image: url("/images/backgrounds/back-0-540.webp");
+    background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
   }
 
   @include mq($until: md) {
-    background-image: url("/images/backgrounds/back-0-720.webp");
+    background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
   }
 
   @include mq($until: lg) {
-    background-image: url("/images/backgrounds/back-0-960.webp");
+    background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
   }
 
   @include mq($until: xl) {
-    background-image: url("/images/backgrounds/back-0-1140.webp");
+    background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
   }
 }
 
 .no-webp .jumbo .jumbo-cover {
-  background-image: url("/images/backgrounds/back-0.jpg");
+  background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
 
   @include mq($until: sm) {
-    background-image: url("/images/backgrounds/back-0-540.jpg");
+    background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
   }
 
   @include mq($until: md) {
-    background-image: url("/images/backgrounds/back-0-720.jpg");
+    background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
   }
 
   @include mq($until: lg) {
-    background-image: url("/images/backgrounds/back-0-960.jpg");
+    background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
   }
 
   @include mq($until: xl) {
-    background-image: url("/images/backgrounds/back-0-1140.jpg");
+    background-image: url("/images/backgrounds/bsidestlv2025-new-date-banner.png");
```

Comment:
These should not be replaced like this
There's a script that generates optimized images for different resolutions